src/gt4py/next/config.py

```diff
@@ -113,3 +113,8 @@ def env_flag_to_int(name: str, default: int) -> int:
 #: The default for whether to allow jit-compilation for a compiled program.
 #: This default can be overriden per program.
 ENABLE_JIT_DEFAULT: bool = env_flag_to_bool("GT4PY_ENABLE_JIT_DEFAULT", default=True)
+
+
+#: Extra user-defined compiler arguments to use in compiled backends.
+COMPILE_CXXFLAGS: str | None = os.environ.get("GT4PY_COMPILE_CXXFLAGS", None)
+COMPILE_CUDAFLAGS: str | None = os.environ.get("GT4PY_COMPILE_CUDAFLAGS", None)
```

Comment:
```suggestion
CXXFLAGS: str | None = os.environ.get("CXXFLAGS", None)
CUDAFLAGS: str | None = os.environ.get("CUDAFLAGS", None)
```
my proposal was this, because it will directly work for properly setup cmake projects.

But now I am not sure, because in DaCe we now violate how CMake uses these flags, because DaCe does `-DCMAKE_CXX_FLAGS=...` it should ignore the environment variable.

Reply:
Maybe we just do this hack in the dace part and remove it from the config again.

Reply:
I have now addressed this comment, I have reverted all changes in gt4py config.